plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -307,7 +307,12 @@ def compare_args_ipa(module, args, ipa):  # noqa
         if not (isinstance(args, dict) and isinstance(ipa, dict)):
             raise TypeError("Expected 'dicts' to compare.")
 
-        for key in args.keys():
+        if ignore is None:
+            ignore = []
+
+        filtered_args = [k for k in args.keys() if k not in ignore]
```

Comment:
It is better not to use ".keys()" here, please, change it to `[key for key in args if key not in ignore]`.

The main reason is that we are trying to enable pylint verification, and it wil fail if kept the way it is now.